barman/server.py

```diff
@@ -1875,6 +1876,8 @@ def get_required_xlog_files(
         :param target_time: target time, in epoch
         :param target_xid: target transaction ID
         :param target_lsn: target LSN
+        :param target_immediate: target that ends recovery as soon as
+            consistency is reached. Defaults to `False`.
```

Comment:
```suggestion
            consistency is reached. Defaults to ``False``.
```

Sphinx recommends double backticks for literals.